server/daemon/src/main.rs

```diff
@@ -874,7 +875,12 @@ async fn kanidm_main(config: Configuration, opt: KanidmdParser) -> ExitCode {
             commands: DbCommands::Backup(bopt),
         } => {
             info!("Running in backup mode ...");
-            backup_server_core(&config, &bopt.path);
+            let backup = bopt
+                .compression
+                .clone()
+                .map(BackupCompression::from)
+                .unwrap_or_default();
```

Comment:
Fall back to the option in the configuration if present?

Reply:
Yeah it'll be irrelevant when I use the config option